docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:8u181-jdk-stretch
+
+LABEL MAINTAINER=fx19880617@gmail.com
+
+ARG PINOT_BRANCH=develop
+RUN echo "Trying to build Pinot on branch ${PINOT_BRANCH}"
+ENV PINOT_HOME=/opt/pinot
+
+# extra dependency for running launcher
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		vim wget curl git maven\
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/winedepot/pinot.git /opt/incubator-pinot  && \
+    cd /opt/incubator-pinot && \
+    git checkout ${PINOT_BRANCH} && \
+    mvn install package -DskipTests -Pbin-dist && \
+    mkdir -p ${PINOT_HOME}/configs && \
+    mkdir -p ${PINOT_HOME}/data && \
+    cp -r pinot-distribution/target/apache-pinot-incubating-*-bin/apache-pinot-incubating-*-bin/* ${PINOT_HOME}/. && \
+    chmod +x ${PINOT_HOME}/bin/*.sh && \
+    mvn dependency:purge-local-repository -DactTransitively=false -DreResolve=false --fail-at-end && \
+    rm -rf /opt/incubator-pinot
```

Comment:
/opt/incubator-pinot is a temp dir right. lets name add a prefix to indicate that this is a staging/build directory.

Also, create a variable for this variable. I see it repeated in multiple places.

Reply:
Remove incubator from the folder name.